release-notes.md

```diff
@@ -12,30 +12,44 @@ can view active developments on GitHub at any time.
 
 ### What to expect from our next release
 
-The next release of TimescaleDB will include basic
+The next release of TimescaleDB, v1.7, will include basic
 [PostgreSQL 12](https://www.postgresql.org/about/news/1976/)
-support and Union Views.
-
-With Union Views, users will be able to query data
-from a continuous aggregate (which will contain data
-based on when the last update job ran). This feature
-will fetch the most updated data from the underlying
-table in order to satisfy your query.
-
-We expect to release these capabilities late in Q1
-of 2020.
+support and a new TimescaleDB capability called *Just-In-Time Aggregates*.
+
+With Just-In-Time Aggregates, users will be able to leverage TimescaleDB's
+precomputed continuous aggregates while still getting a real-time view over the
+latest data.
+
+In particular, continous aggregates made it really fast to query data:
+precompute some aggregate (such as the min/max/average value over each
+hour). This way, if you are collecting raw data every seconds, to query hourly
+data over the past week means reading 24*7=168 values from the database, as
```

Comment:
escape all of the asterisks here and line below to prevent odd visuals:
![Screen Shot 2020-02-14 at 12 45 54 PM](https://user-images.githubusercontent.com/12288273/74554633-f9b61500-4f27-11ea-9ddb-0a253755b3c3.png)

Reply:
or switch to 'x'